stable/database/templates/statefulset.yaml

```diff
@@ -63,10 +63,21 @@ spec:
       - name: init-disk
         image: {{ template "init.image" . }}
         imagePullPolicy: {{ default "" .Values.busybox.image.pullPolicy | quote }}
-        command: ['chmod' , '770', '/var/opt/nuodb/archive', '/var/log/nuodb']
+        command:
+          - 'chmod'
+          - '770'
+          - '/var/opt/nuodb/archive'
+          - '/var/log/nuodb'
```

Comment:
@NikTJ777 says:
Wherever it lands, there are a few points on the change:

the new code seems to no longer chmod on /var/log/nuodb
1a and this chmod needs to be 'chmod -R'
the path 'var/opt/nuodb/backup' should be '/var/opt/nuodb/backup'

Reply:
the section I commented above relates to the non-HC SM. I will fix the -R here too.